app/src/test/java/org/simple/clinic/teleconsultlog/teleconsultrecord/screen/TeleconsultRecordUiRendererTest.kt

```diff
@@ -63,6 +64,51 @@ class TeleconsultRecordUiRendererTest {
     verify(ui).setPatientTookMedicines(Yes)
     verify(ui).setPatientConsented(Yes)
     verify(ui).renderPatientDetails(patient)
+    verify(ui).hideDoneProgress()
+    verifyNoMoreInteractions(ui)
+  }
+
+  @Test
+  fun `when teleconsult record create state is creating, then show done progress`() {
+    // given
+    val patientUuid = UUID.fromString("822e1457-1c2a-46de-8a07-e8dca3597c50")
+    val model = TeleconsultRecordModel
+        .create(
+            patientUuid = patientUuid,
+            teleconsultRecordId = UUID.fromString("9c6e2931-0f33-4560-92a3-8a08865e8103")
+        )
+        .creatingTeleconsultRecord()
+
+    // when
+    uiRenderer.render(model)
+
+    // then
+    verify(ui).setTeleconsultationType(Audio)
+    verify(ui).setPatientTookMedicines(Yes)
+    verify(ui).setPatientConsented(Yes)
+    verify(ui).showDoneProgress()
+    verifyNoMoreInteractions(ui)
+  }
+
+  @Test
+  fun `when cloning prescriptions is completed, then hide progress`() {
```

Comment:
It is an implementation detail that cloning the prescriptions completes the recording of a teleconsult. We can rename this to `when the teleconsult record is created, then hide the progress`.